packages/core/src/analytics/integrations/Omnitracking/definitions.js

```diff
@@ -523,6 +524,88 @@ export const trackEventsMapper = {
     wishlistId: data.properties?.wishlistId,
     lineItems: getProductLineItems(data),
   }),
+  [eventTypes.PRODUCT_CLICKED]: data => ({
+    tid: 2926,
```

Comment:
I think we could also map the productId here, like in the PRODUCT_UPDATED mapping. what do you think?

Reply:
ProductId field is inside of lineItems property. 
LineItems it's a json string, with product clicked information.

Reply:
Added product id to event track property list.